js/profile.js

```diff
@@ -0,0 +1,263 @@
+$(document).ready(function(){
+
+    /**
+     * Globals
+     */
+    // Div containing the image
+    var profileImage = $("#image-profile");
+    // jCrop obj to return
+    var jCropAPI;
+
+    // Custom right click bindings
+    $("#profile-picture").bind("contextmenu", function(e){
+
+        // Stop the default menu from activating
+        e.preventDefault();
+
+        // Display the new menu
+        $("#_contextmenu-profile-picture").finish().toggle(100).
+            css({
+                top: e.pageY,
+                left: e.pageX
+            });
+    });
+
+    // Opposite binds! (Close the context menus when the user clicks elsewhere)
+    $(document).bind("mousedown", function(e){
+
+        // Make sure the click wasn't inside a context menu
+        if(!$(e.target).parents("._contextmenu").length > 0){
+            // So we're here, that means we can hide all context menus
+            $("._contextmenu").hide(100);
+        }
+    });
+
+    // Context menu click assignments here
+    $("._contextmenu li").click(function(){
+
+        var avatar;
+        avatar = $("#profile-picture").attr("src");
```

Comment:
why is the assignment split off from the creation here? Also, we should use "let" not "var". var is the old way of doing things in javascript and it is a bit broken :-)